src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ReaderServiceImplTest {
+    private static final String PATH_TO_FILE = "src/resources/testInput.csv";
+    private static final String FAKE_PATH_TO_FILE = "src/resourses/testFile12.csv";
+
+    private static ReaderService readerService;
+
+    @BeforeAll
+    static void setUp() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
+    void readFromFile_pathToFile_ok() {
+        List<String> expectedLines = List.of(
+                "b,banana,20",
+                "b,apple,100",
+                "s,banana,100",
+                "p,banana,13",
+                "r,apple,10",
+                "p,apple,20",
+                "p,banana,5",
+                "s,banana,50"
+        );
+        List<String> lines = readerService.readFromFile(PATH_TO_FILE);
+        assertEquals(expectedLines, lines);
+    }
```

Comment:
I guess it's better to use words "expected"/"actual" intead of "expectedLines"/"lines", or at least "expectedLines"/"actualLines".

Reply:
agree with @Alexsandr-Yablunovskyi

Reply:
LOL, I renamed all variables beside except where it says